plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -188,11 +188,179 @@ def __cmp__(self, other):
 
 
 class Distribution(UnitMixin, FileContentUnit):
-    # TODO add docstring to this class
+    """
+    Model for an RPM distribution tree (also sometimes referenced as an installable tree).
+    A distribution tree is described by a file in root of an RPM repository named either
+    "treeinfo" or ".treeinfo". This INI file is used by system installers to boot from a URL.
+    It describes the operating system or product contained in the distribution tree and
+    where the bootable media is located for various platforms (where platform means
+    'x86_64', 'xen', or similar).
+
+    Note: This model should be rewritten. It fails to capture what a distribution tree
```

Comment:
https://pulp.plan.io/issues/1769

I assumed it was going to come up in the very near future since it probably blocks https://pulp.plan.io/issues/764, but maybe not. I'll make a note of the issue in the docblock.